Lib/fontTools/ufoLib/converters.py

```diff
@@ -6,16 +6,16 @@
 
 # adapted from the UFO spec
 
-def convertUFO1OrUFO2KerningToUFO3Kerning(kerning, groups):
+def convertUFO1OrUFO2KerningToUFO3Kerning(kerning, groups, glyphSet=set()):
```

Comment:
Using a mutable as default argument is adviced against in Python.

Reply:
good point. An empty tuple `()` or `frozenset()` will do

Reply:
even though it is not modified, but only read by the function. still a good idea